vendor/bat-native-ads/src/bat/ads/internal/account/account_unittest.cc

```diff
@@ -548,11 +551,13 @@ TEST_F(BatAdsAccountTest, DepositForCash) {
 
   transactions::GetForDateRange(
       DistantPast(), DistantFuture(),
-      [&expected_transactions](const bool success,
-                               const TransactionList& transactions) {
-        ASSERT_TRUE(success);
-        EXPECT_EQ(expected_transactions, transactions);
-      });
+      base::BindOnce(
+          [](TransactionList expected_transactions, const bool success,
```

Comment:
`The parameter 'expected_transactions' is copied for each invocation but only used as a const reference; consider making it a const reference`

Reply:
Unfortunately making const reference here doesn't prevent copying of an object. The good news is that
`expected_transactions` is moved below `std::move(expected_transactions)` so there is no copy.
But I think I will change parameter type to `const TransactionList&` as it will be more readable.